src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -46,6 +46,9 @@
     @Argument(fullName="createOutputBamIndex", shortName="createOutputBamIndex", doc = "If true, create a BAM/CRAM index when writing a coordinate-sorted BAM/CRAM file.", optional=true)
     public boolean createOutputBamIndex = true;
 
+    @Argument(fullName="createMD5", shortName="createMD5", doc = "If true, create a MD5 digest for any BAM/SAM/CRAM file creates.", optional=true)
```

Comment:
We should be consistent about whether the arg names should include the output type or  not. We used "createOutputBamIndex"  for the index, but "createMD5" for MD5. Likewise, in https://github.com/broadinstitute/gatk/commit/2fb990dfeb7f5be20d6d09e184327d3872094f26#diff-42a5b252e9a7ec8c16858feb707afda3 (not yet reviewed), we added the same args for VCFs but used output-format-specific names.  We could go wither way, but we should be consistent (for many tools it won't matter, but some like HC output a vcf and a bam).